src/AppInstallerCLITests/TestData/InstallFlowTest_WindowsFeatures.yaml

```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: AppInstallerCliTest.TestExeInstaller.WindowsFeatures
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+PackageName: AppInstaller Test Installer
+ShortDescription: Installs exe installer with invalid Windows Features dependencies
+Publisher: Microsoft Corporation
+Moniker: AICLITestZip
+License: Test
+Installers:
+    - Architecture: x86
+      InstallerUrl: https://ThisIsNotUsed
+      InstallerType: exe
+      InstallerSha256: 65DB2F2AC2686C7F2FD69D4A4C6683B888DC55BFA20A0E32CA9F838B51689A3B
+      InstallerSwitches:
+        Custom: /custom /scope=machine
+        SilentWithProgress: /silentwithprogress
+        Silent: /silence
+        Update: /update
+      Dependencies:
+        WindowsFeatures:
+        - netfx3

```

Comment:
Does casing matter for windows features? If it does, it probably wouldn't be a bad idea to have a test that validates incorrect casing fails.

Reply:
casing doesn't matter, added tests to verify that